projects/idea-hub-smart-contract-solidity/Readme.md

```diff
@@ -0,0 +1,5 @@
+IdeaHub Smart Contract
+===========================
+[IdeaHub Interface](contracts/IIdeaHub.sol)
```

Comment:
Could you please add some descriptions to this Readme regarding the contract design and structure?